src/main/java/com/ecommerce/ECommerceApp/model/Payment.java

```diff
@@ -20,18 +22,18 @@ public void setPaymentID(int PaymentID){
         this.paymentID = paymentID;
     }
 
-    public PaymentType getPaymetType(){
+    public PaymentType getPaymentType(){
         return paymentType;
     }
 
-    public void SetPaymentType(PaymentType paymentType){
+    public void setPaymentType(PaymentType paymentType){
         this.paymentType = paymentType;
     }
 
     public PaymentStatus getPaymentStatus(){
         return paymentStatus;
     }
-    public void SetPaymentStatus(PaymentStatus status){
+    public void setPaymentStatus(PaymentStatus status){
         this.paymentStatus = status;
     }
```

Comment:
Fixed typo in getPaymentType() for readability, made other chances to follow pascal-case instead of CamelCase which was used (which is meant for class names, not method names). These changes will allow for more consistent naming and reduce need to relearn naming schemes when code is being viewed.

Source: https://www.tutorialspoint.com/java/java_documentation.htm